code/go/0chain.net/smartcontract/minersc/handler.go

```diff
@@ -406,10 +406,16 @@ type nodeStat struct {
 //	484:
 func (mrh *MinerRestHandler) testNodeStat(w http.ResponseWriter, r *http.Request) {
 	var (
-		id               = r.URL.Query().Get("id")
+		providerId      = r.URL.Query().Get("provider_id")
```

Comment:
Since we have new param `provider_type`, so I think use `provider_id` would be better to match it.

Reply:
As this is an extension of the nodeStat endpont so it should use the same parameters. Hense use `id` instead of `provider_id` and no `provider_type` field.

We should keep our endpoint parameters self-consistent where possible. The only effect here is to make our parameters inconsistent, with no effect except to confuse users.